test/tf_exec.cpp

```diff
@@ -131,4 +131,46 @@ TEST(tf_exec, Op_RealDiv) {
 
 }
 
+TEST(tf_exec, Op_Square) {
```

Comment:
C++ Unit test - nice!
(We will have to start reorganizing these tests - but that's a separate PR)